### PR TITLE
Creation of function that generates User Response Data 

### LINKED ISSUE #76: Add userPath Answers for Form info for submission

### PR DESCRIPTION
**Pull request by vlbee**
closes #76 

Upon hitting submit on the form, need to send both the user form data and a log of their answers to the BE. This function generates the response log, based on the usersPath from the Wayfinder state.

### CODE REVIEW

File: src/components/form/form.js [hunk 1]
```diff
@@ -15,6 +16,31 @@ export default class Form extends React.Component {
     };
   }
 
+  // Loops through the usersPath using the next key in the array as a reference to generate the original
+  // Question and Answer from the Wayfinder content data object
+  // Returns array of Question and Answer objects (eliminating 'continue' answers)
+  generateUserResponseDataFromPath = () => {
```
Comment (jennah2121) [inline src/components/form/form.js]: I think there is a bug here. Where the legal orders are concerned, the question and answer data is not updated correctly if the user goes back as they have the same goto in wayfinder data file. 

Also it might be worth preventing the user from changing their choices after they submit the form as this could potentially result in duplicated forms for GP. I will write an issue about this as only the bug needs to be addressed before merge.

Comment (vlbee) [reply src/components/form/form.js]: I've updated wayfinderData.js so that SGO/CAO answers now have separate keys - this was causing the bug. 

We will also need a unit test for this function to catch edge cases.
